src/pipeline/src/etl/transform/transformer/greptime.rs

```diff
@@ -47,6 +51,25 @@ pub struct GreptimeTransformer {
     schema: Vec<ColumnSchema>,
 }
 
+/// Parameters that can be used to configure the `greptime_identity` pipeline.
+#[derive(Debug, Clone, Default)]
+pub struct GreptimeIdentityPipelineParams {
+    /// Whether to flatten the JSON object.
+    pub flatten_json_object: bool,
+}
+
+impl GreptimeIdentityPipelineParams {
+    /// Create a `GreptimeIdentityPipelineParams` from the HTTP headers.
+    pub fn from_headers(headers: &HeaderMap) -> Self {
+        Self {
+            flatten_json_object: headers
```

Comment:
Don't use one header for one parameter, just use `x-greptime-pipeline-params` as the header key and make the values in the form of  query string such as ''a=b&b=c`  like that.